CMakeLists.txt

```diff
@@ -240,7 +241,8 @@ if("${HIP_COMPILER}" MATCHES "hcc")
   endif()
 endif()
 
-target_link_libraries(rccl PRIVATE hip::device dl)
+target_include_directories(rccl PRIVATE ${ROCM_PATH}/rocm_smi/include)
+target_link_libraries(rccl PRIVATE hip::device dl -lrocm_smi64 -L${ROCM_PATH}/rocm_smi/lib)
```

Comment:
does rocm_smi64 not have a cmake target?

Reply:
Saad will create ticket for rocm-smi-lib to support cmake target